std/haxe/macro/Expr.hx

```diff
@@ -26,7 +26,11 @@ extern enum Position {
 }
 #else
 /**
-	Represents a position in a file.
+	Represents the position of where a given type, field, function or variable
+	is defined. This is used by the compiler to display for example errors 
+	or to provide code completion information.
```

Comment:
I don't like the listing here, it's not exhaustive and maintaining it to be exhaustive would be annoying and pointless.